pigLatin/index.js

```diff
@@ -0,0 +1,34 @@
+const translateToPigLatin = (text) => {
+    if (!text) {
+        return "";
+    }
+    const words = text.split(/(\s+)/);
```

Comment:
What are you trying to do with this regex? I think you are trying to split a string that holds multiple words. To do this, I usually use: 

`text.split(" ");`

This will return an array of words without the spaces between. It is kind of a companion method to .join(" ") that you use on line 7.